.circleci/config.yml

```diff
@@ -64,13 +64,21 @@ jobs:
     executor: node-erbium-executor
     steps:
       - checkout_from_workspace
+      - add_ssh_keys:
+        fingerprints:
```

Comment:
You're getting a build error I think around this line

Reply:
Yeah, investigating why the config error, since this is [how the docs describe it](https://circleci.com/docs/2.0/configuration-reference/#add_ssh_keys)